apps/server/src/migrations/mikro-orm/Migration20240304123509.ts

```diff
@@ -0,0 +1,68 @@
+/* istanbul ignore file */
+import { Migration } from '@mikro-orm/migrations-mongodb';
+
+export class Migration20240304123509 extends Migration {
+	async up(): Promise<void> {
+		const columBoardResponse = await this.driver.nativeUpdate(
+			'boardnodes',
+			{ type: 'column-board' },
+			{ $set: { isVisible: true } }
```

Comment:
Maybe just a thought about the naming of the property: As far as I understand we want to persist the visibility to others and not the general cisibility of the node. Wouldn't be a name like `published` more appropriate? Also I'm not sure if we should use the `is` (boolean) prefix in the database.

Reply:
I did it this was because the endpoint is called visibility... I would avoid changing it now